### ISSUE TITLE
vp9 2 pass (first pass could be speed up)

### ISSUE THREAD

**Issue by Tupsi** [cdgriffith/FastFlix#194]
google suggest speeding up the first pass always when using 2-pass. So I would suggest giving the speed option a fixed 4 instead what you want for the 2nd pass to work.

https://developers.google.com/media/vp9/settings/vod/

Or if thats not to much work, make it adjustable as a profile setting.

**Comment by cdgriffith** [comment]
Thanks for the link, never saw this guide before so I will go through it and see what I can add / adjust!

**Comment by Tupsi** [comment]
also I just realised you kinda force people to use .webm with vp9. If you want to preserve chapters you must not do that and stay at .mkv. Last time I checked (which granted was a while ago) webm did not support chapters, so the whole point of having "copy chapters" checked and using .webm should not work. Well maybe it works (as its basically a stripped down mkv container), but it is not part of the standard .webm format.
